lazy_tensor_core/lazy_tensor_core/csrc/init_python_bindings.cpp

```diff
@@ -578,9 +567,21 @@ void InitLtcModuleBindings(py::module m) {
           return result_tuple;
         });
   m.def("_ltc_set_default_device", [](const std::string& device) {
-    return SetCurrentThreadDevice(device);
+    // TODO: Replace this API with _ltc_set_default_device_type.
+    // The reasons why deprecating this API are that:
+    // i) It makes sense to set default device type to CPU, GPU or TPU,
+    // but not lazy given that would just use whatever default hardware type.
+    // ii) Setting ordinal like lazy:0 doesn't make any sense now as distributed
+    // training is still under development.
+    LOG(ERROR) << "Setting the default device is deprecated. Use "
+                  "_ltc_set_default_device_type to set the default "
+                  "device type instead.";
+    return;
+  });
+  m.def("_ltc_get_default_device", []() {
+    // It's always lazy:0 given distributed training is not supported yet.
+    return "lazy:0";
```

Comment:
I assume if pytorch/xla has fully adopted LTC, they would just use `lazy` device instead of `xla` device. Also, the previous logic won't return `xla:0` as well given the bridge will always use the lazy keyword: https://github.com/pytorch/pytorch/blob/lazy_tensor_staging/lazy_tensor_core/lazy_tensor_core/csrc/aten_ltc_bridge.cpp#L263.

Reply:
ok. this is probably TBD in terms of how fast xla makes the change, or if they consider keeping xla:: device.

But I still prefer to merge as-is and then revisit this once we're focusing more on e2e design of the UI and pybindings.

Reply:
Let me also add a TODO here suggest that it's deprecated like the above `_ltc_set_default_device`.